File: covid-19-backend/CovidHelp/Models/ViewModels/InfectedStatisticsViewModel.cs
using NetCoreCommons.Shared.Helpers;
using NetCoreCommons.Shared.Models;
using System;

namespace CovidHelp.Models.ViewModels
{
  public class InfectedStatisticsViewModel : BaseViewModel<long>
  {
    public long CountryId { get; set; }
    // mapped from mapper
    public string CountryName { get; set; }
    public GeoPointViewModel CountryLocation { get; set; }
    public DateTime Date { get; set; }
    public long Confirmed { get; set; }
    public long Deaths { get; set; }
    public long Recovered { get; set; }
    public override string OptionText => $"{CountryName} {FormattingHelper.GetFormattedDate(Date)}";
  }
}

File: covid-19-backend/CovidHelp/angular/src/app/generated-api/models/infected-statistics-view-model.ts
/* tslint:disable */
import { GeoPointViewModel } from './geo-point-view-model';
export interface InfectedStatisticsViewModel {
  recovered?: number;
  countryId?: number;
  countryLocation?: GeoPointViewModel;
  date?: string;
  confirmed?: number;
  deaths?: number;
  countryName?: string;
  optionText?: string;
  id?: number;
  guid?: string;
  optionId?: string;
  isDeleted?: boolean;
  optionAdditionalInfo?: {[key: string]: string};
}
